modules/launcher/launcher-impl/src/main/resources/META-INF/config/system.properties

```diff
@@ -69,3 +69,8 @@ xp.config.paths=${xp.home}/config
 #
 # xp.init.maxTries = 300
 
+#
+# Global security settings
+#
+# xp.disallow.access.to.draft.branch.for.anonymous = true
```

Comment:
It should be a portal config. Not a global config.
Config should specify principals who have access